README.md

```diff
@@ -55,22 +55,19 @@ module.exports = {
             {
               name: 'description',
               content: 'mini-html-webpack-template'
-            },
-            {
-              property: 'og:description',
-              content: 'mini-html-webpack-template'
-            }
-          ],
-          links: [
-            {
-              rel: 'icon',
-              type: 'image/x-icon',
-              href: 'https://assets-cdn.github.com/favicon.ico'
             }
           ]
         },
         body: {
           raw: '<div id="root"></div>'
+        },
+        attrs: {

```

Comment:
I'm not really sure what HTML this will produce. Why is it an array?

Reply:
From the tests, this producing valid HTML. Say by default `mini-html-webpack-plugin` prints webpack output script tags as `<script src="/build.js"></script>`, so this feature adds support for userland attributes like `defer` or [others](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) and for the readme example output would be `<script async="" type="text/javascript" src="/build.js"></script>`.

I've maked this an array for keeping current api consistence.